packages/host/src/components/Login/Login.tsx

```diff
@@ -1,20 +1,17 @@
 import React from 'react';
 import {
   ArrowRightIcon,
-  Box,
-  Stack,
-  Typography,
-  Autocomplete,
   useTranslation,
   AutocompleteRenderInputParams,
-  defaultOptionMapper,
   LoadingButton,
+  Box,
```

Comment:
I was getting a little confused with the jsx. These complicated layouts can get so annoying! I had an idea - it's the last commit so I can revert if it doesn't help.

Other than `LoginLayout`, I also started using `StoreSearchInput`